include/ringmesh/basic/types.h

```diff
@@ -73,8 +73,15 @@ namespace RINGMesh
     using vecn = GEO::vecng< DIMENSION, double >;
     // This is an array of 3 doubles
     using vec3 = vecn< 3 >;
-    // This is an array of 3 doubles
+    // This is an array of 2 doubles
     using vec2 = vecn< 2 >;
+    // This is an array template of signed integers
+    template < index_t DIMENSION >
+    using intvecn = GEO::vecng< DIMENSION, signed_index_t >;
```

Comment:
This is an open question: intvecn seems a long type, could vecin be clear enough?

Reply:
Well, since i use intvecn, intvec3 and intvec2,  i tried to keep the same nomenclature than vecn, vec3, vec2, and did it like that to avoid typos if we put only ivecn, ivec3, ivec2 ... As you wish, but thought it was clearer this way.

Reply:
I would prefer ivecn